examples/deferred_processing/automated_deferred_processing/README.md

```diff
@@ -0,0 +1,348 @@
+to a file with a `jpatch` extension.
+
+```
+sed -E -f commands.sed json-patch.template | \
+    tr '\n' ' ' > json-patch.jpatch
+```
+
+.NOTE
+The prior step is in the Makefile
+in the  `json-patch.jpatch` target.
+
+#### Adding the patch and using it in the pipeline
+To add the json patch to our pipeline,
+we can just paste it in,
+or run the Unix command below to add it
+to an otherwise valid JSON pipeline spec 
+in the file `branch-mover.pipeline`
+
+```
+jq ".pod_patch=\"$(cat json-patch.jpatch)\"" branch-mover.pipeline \
+        > branch-mover.json
+```
+
+That file, `branch-mover.pipeline`, includes one line
+that uses the `PACHYDERM_AUTH_TOKEN` environment variable
+to authenticate to Pachyderm. 
+
+```
+          "echo ${PACHYDERM_AUTH_TOKEN} | pachctl auth use-auth-token"
+```
+
+That line is inserted prior to creating the branch, 
+making the pipeline transform in `branch-mover.pipeline`
+look like this:
+
+```
+"transform": {
+      "cmd": ["sh" ],
+      "stdin": [
+          "echo '{\"pachd_address\": \"grpc://pachd:650\"}' | pachctl config set context default --overwrite",
+          "echo ${PACHYDERM_AUTH_TOKEN} | pachctl auth use-auth-token",
+          "pachctl create branch edges_dp@master --head dev"
+      ],
+    "image": "pachyderm/pachctl:1.11.0"
+  }
+```
+
+.NOTE
+The prior step is in the Makefile
+in the  `branch-mover.json` target.
+
+#### Create the pipeline
+
+Finally, create the pipeline using that spec
+
+```
+pachctl create pipeline -f branch-mover.json
+```
+
+.NOTE
+The prior step is in the Makefile
+in the  `create-branch-mover` target.
+
+## Example run-through
+
+This example can be used with no authentication or with authentication activated.
+The only difference is the command you use to create the pipeline 
+in the second step, below.
+
+
+1. If the DAG
+   used by the deferred processing example
+   hasn't yet been created,
+   create that starting DAG 
+   by running this command 
+   from inside this directory.
+   
+   ```
+   make create-deferred-processing-cluster
+   ```
+
+2. If you are not using authentication,
+   create the branch-mover cron pipeline
+   using the `create-branch-mover-no-auth` makefile target.
+   
+   ```
+   make create-branch-mover-no-auth
+   ```
+
+   If you have authentication activated,
+   create the branch-mover cron pipeline
+   using the `create-branch-mover` makefile target.
+   
+   ```
+   make create-branch-mover
+   ```
+
+3. Watch pachctl jobs in another terminal window
+   using this command. 
+   
+   ```
+   watch -cn 2 pachctl list job --no-pager
+   ```
+
+.NOTE On macOS, you may need to install `watch`, 
+which may be installed via [Homebrew](https://brew.sh/)
+using the command `brew install watch`
+   
+4. Every minute, you should see a job triggered on `branch-mover`.
+
+
+5. Commit data to the `images_dp_1` repo.
+
+```
+pachctl put file images_dp_1@master:1VqcWw9.jpg -f http://imgur.com/1VqcWw9.jpg
+```
+
+6. A job will be triggered on `edges_dp`, 
```

Comment:
This looks like it's not a step but a description of the result for the previous step. So I suggest removing the numbered list formatting from this paragraph.